### PR TITLE
Explicit error when stream_id is not found

### PR DESCRIPTION
**Pull request by lmcarril**

### CODE REVIEW

File: s64da_benchmark_toolkit/streams.py [hunk 1]
```diff
@@ -101,7 +101,11 @@ def run(self):
     def get_stream_sequence(self, stream_id):
         streams_path = os.path.join(self.benchmark.base_dir, 'queries', 'streams.yaml')
         with open(streams_path, 'r') as streams_file:
-            return yaml.load(streams_file, Loader=yaml.Loader)[stream_id]
+            streams = yaml.load(streams_file, Loader=yaml.Loader)
+            print(streams)
```
Comment (sdressler) [inline s64da_benchmark_toolkit/streams.py:105]: Is the print intentional?

File: s64da_benchmark_toolkit/streams.py [hunk 2]
```diff
@@ -101,7 +101,11 @@ def run(self):
...
+            if stream_id not in streams:
+                raise Exception('Stream file {} does not contain stream id {}'.format(streams_path, stream_id))
```
Comment (sdressler) [inline s64da_benchmark_toolkit/streams.py:107]: Please use a `ValueError`, and maybe build a f-string.

File: s64da_benchmark_toolkit/streams.py [hunk 3]
```diff
@@ -101,7 +101,11 @@ def run(self):
...
+            return streams[stream_id]
```
Comment (sdressler) [inline s64da_benchmark_toolkit/streams.py:108]: Thinking about:

    sid = streams.get(stream_id)
    if not sid:
        raise ...

Comment (lmcarril) [reply s64da_benchmark_toolkit/streams.py:108]: As you prefer, although it seems more explicit that if the key does not exist the error is raised.

Comment (sdressler) [reply s64da_benchmark_toolkit/streams.py:108]: Totally fine, was just a thought really. Python even encourages

     try:
         return streams[stream_id]
     catch KeyError as exc:
         ...

Your call, no pref here. Explicit is good.

File: s64da_benchmark_toolkit/streams.py (hunk 1 above)
Comment (lmcarril) [reply s64da_benchmark_toolkit/streams.py]: It was intentional for testing :P.
Removed

File: s64da_benchmark_toolkit/streams.py (hunk 2 above)
Comment (lmcarril) [reply s64da_benchmark_toolkit/streams.py]: Done

**Pull request merged**